content/blog/findings-when-coverting-tests-to-pester5.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "Findings when converting tests to Pester 5"
+date: 2020-05-17T05:00:00+01:00
+type: "post"
+draft: false
+author: johlju
+---
+
+If you have any questions or suggestions around this blog post then please
+reach out to `@johlju` in the Virtual PowerShell User Group [#DSC channel](https://dsccommunity.org/community/contact/).
+
+## Pester
+
+Pester is used throughout the DSC modules and their common modules to ensure
```

Comment:
suggestion:
"[Pester](https://github.com/pester), the famous DSL and module in the PowerShell community, is used ..."

Reply:
note the link, btw